INSTALL.md

```diff
@@ -1,143 +1,120 @@
-## Installation Guide
-
-### **Instruction to install PISA on your local machine**
-
-1. Install the essential dependencies
-
-   1. Anaconda (https://www.anaconda.com/)
-      ```bash
-      wget -nc https://repo.anaconda.com/archive/Anaconda3-2023.07-2-Linux-x86_64.sh
-
-      bash Anaconda3-2023.07-2-Linux-x86_64.sh
-      ```
-   2. pip (Python package installer)
-      ```bash
-      conda install pip
-      ```
-   3. git
-      ```bash
-      sudo apt install git
-      ```
-
-   Other required libraries (listed below) will be installed automatically during the setup (listed in `setup.py`).
-
-   - fast-histogram >= 0.10
-   - h5py
-   - iminuit >= 2
-   - line_profiler
-   - llvmlite
-   - matplotlib >= 3.0
-   - nlopt
-   - numba >= 0.53
-   - numpy >=1.17, < 1.23
-   - pandas
-   - pint <=0.19
-   - py-cpuinfo
-   - pyarrow
-   - scikit-learn <= 1.1.2
-   - scipy >= 1.6
-   - simplejson == 3.18.4
-   - sympy
-   - tables
-   - tabulate
-   - tqdm
-   - uncertainties
-   - kiwisolver >= 1.3.1
-   - cycler >= 0.10
-   - packaging >= 20.0
-   - fonttools >= 4.22.0
-   - python-dateutil >= 2.7
-   - pillow >= 8
-   - contourpy >= 1.0.1
-   - pyparsing >= 2.3.1
-   - threadpoolctl >= 2.0.0
-   - joblib >= 1.0.0
-   - numexpr
-   - pytz >= 2020.1
-   - tzdata >= 2022.1
-   - mpmath >= 0.19
-   - blosc2 >= 2.3.0
-   - future
-   - ndindex >= 1.4
-   - msgpack
-   - six >= 1.5
-2. Create conda environment/ virtual environment for the installation of PISA. Assume that the name of the environment is `pisa_env` you can choose your preferred version of python >= 3.10
+# Installation Guide
+
+
+## Instruction to install PISA using Anaconda or Miniconda
+
+The following commands are intended for execution in a terminal. If you encounter any problems with the installation, please create an issue and/or ask on the IceCube slack (if possible).
```

Comment:
Mention that commands assume bash shell (see https://github.com/icecube/pisa/blob/f8ce002424791a273c4bdea986a501e07418e25b/INSTALL.md).